src/server/graphql/mutations/updateReflectionLocation.js

```diff
@@ -45,15 +62,39 @@ export default {
     if (!phaseItem || phaseItem.teamId !== teamId) return sendPhaseItemNotFoundError(authToken, retroPhaseItemId);
     if (!phaseItem.isActive) return sendPhaseItemNotActiveError(authToken, retroPhaseItemId);
 
+    if (reflectionGroupId) {
+      const reflectionGroup = await dataLoader.get('retroReflectionGroups').load(reflectionGroupId);
+      if (!reflectionGroup) return sendReflectionGroupNotFoundError(authToken, reflectionGroupId);
+      if (reflectionGroup.meetingId !== meetingId) return sendReflectionGroupNotFoundError(authToken, reflectionGroupId);
+    }
     // RESOLUTION
     await r.table('RetroReflection').get(reflectionId)
       .update({
         sortOrder,
+        reflectionGroupId,
         retroPhaseItemId,
         updatedAt: now
       });
+    const orphanedReflectionId = getOrphanedReflectionId(reflectionGroupId, oldReflectionGroupId);
+    const removedGroupId = orphanedReflectionId ? oldReflectionGroupId : undefined;
+    if (orphanedReflectionId) {
+      await r({
```

Comment:
same deal, I suspect it's ok? No` r.expr`?

Reply:
yep! r is shorthand for r.expr